src/Symfony/Component/Validator/Constraints/Image.php

```diff
@@ -72,6 +78,8 @@ class Image extends File
     public $minWidthMessage = 'The image width is too small ({{ width }}px). Minimum width expected is {{ min_width }}px.';
     public $maxHeightMessage = 'The image height is too big ({{ height }}px). Allowed maximum height is {{ max_height }}px.';
     public $minHeightMessage = 'The image height is too small ({{ height }}px). Minimum height expected is {{ min_height }}px.';
+    public $minPixelsMessage = 'The image has to few pixels ({{ pixels }} pixels). Minimum amount expected is {{ min_pixels }} pixels.';
```

Comment:
I think the error message should directly show the problem, so at least show the direct comparison between the set limit and the problematic value. If it helps, maybe there should be some kind of calculation shown. This will make the message more bloated, however. What can be done is that al least the height and width parameters will be set, so custom messages can use these.
If someone has an idea about this, please let us know :)